x-pack/plugins/apm/public/components/shared/charts/Histogram/__test__/Histogram.test.js

```diff
@@ -102,8 +102,9 @@ describe('Histogram', () => {
             transactionId: '99c50a5b-44b4-4289-a3d1-a2815d128192'
           },
           style: { cursor: 'pointer' },
-          x: 869010,
+          center: 869010,
```

Comment:
Good find! Can you perhaps call this `xCenter` or something to denote that it's for the x coordinate (since there is also `y` property on the object).